gammapy/estimators/points/core.py

```diff
@@ -850,3 +873,79 @@ def resample_axis(self, axis_new):
             fluxpoints.append(fp_new)
 
         return self.__class__.from_stack(fluxpoints, axis=axis_new)
+
+
+class LightCurve(FluxPoints):
+    """Specialized 'FluxPoints' heir class for lightcurves"""
+
+    @classmethod
+    def read(cls, filename, **kwargs):
+        kwargs.setdefault("format", "lightcurve")
+        return super().read(filename, **kwargs)
+
+    @classmethod
+    def from_table(cls, table, **kwargs):
+        table = table_standardise_units_copy(table)
+
+        if super()._table_guess_format(table) != "lightcurve":
+            raise ValueError("The input table is not a lightcurve!")
+
+        kwargs.setdefault("format", "lightcurve")
+
+        return super().from_table(table, **kwargs)
+
+    def to_stingray_lightcurve(self):
+        """Transform to a 'stingray.Lightcurve' object.
+        Only objects with a single energy bin are allowed.
+        """
+
+        if self.geom.axes["energy"].nbin > 1:
+            raise ValueError("Only lightcurves with 1 energy bin are supported.")
+
+        try:
+            from stingray import Lightcurve
+
+            counts = self.counts.data
+            axis = self.counts.geom.axes.index_data("dataset")
+            sum_on_dataset = np.nansum(counts, axis=axis).flatten()
+            time = self.geom.axes["time"].center
+            gti = self.gti
+            if gti is not None:
+                gti = np.vstack([gti.time_start.mjd, gti.time_stop.mjd]).T
+            tref = self.geom.axes["time"].reference_time.mjd
+
+            return Lightcurve(time, sum_on_dataset, gti=gti, mjdref=tref)
+
+        except ImportError:
+            raise ImportError("stingray is not installed in the environment.")
+
+    def write(self, filename, **kwargs):
+        format = kwargs.pop("format", None)
+        if format is None:
+            format = "lightcurve"
+        elif format not in ["lightcurve", "binned-time-series"]:
+            raise ValueError(
+                "Requested format is not supported as output for LightCurve"
+            )
+        kwargs.setdefault("format", format)
+        return super().write(filename, **kwargs)
+
+    def to_table(self, **kwargs):
+        format = kwargs.pop("format", None)
+        if format is None:
+            format = "lightcurve"
+        elif format not in ["lightcurve", "binned-time-series"]:
+            raise ValueError(
+                "Requested format is not supported as output for LightCurve"
+            )
+        kwargs.setdefault("format", format)
+        return super().to_table(**kwargs)
+
+    def fvar(self, flux_quantity="flux"):
+        return compute_lightcurve_fvar(self, flux_quantity)
+
+    def fpp(self, flux_quantity="flux"):
+        return compute_lightcurve_fpp(self, flux_quantity)
```

Comment:
Do these functions work correctly for multi-dimensional light curves?
Otherwise, the class can ensure that it has only one energy bin and no additional axes.

Reply:
Yes these functions return tables which can accomodate multi-dimensional objects.